core/eolearn/core/eoexecution.py

```diff
@@ -95,7 +95,7 @@ def _parse_and_validate_execution_args(execution_args: object) -> List[Dict[EONo
         return [input_kwargs or {} for input_kwargs in execution_args]
 
     @staticmethod
-    def _parse_execution_names(execution_names, execution_args: Optional[List[str]]) -> List[str]:
+    def _parse_execution_names(execution_names: object, execution_args: Sequence) -> List[str]:
```

Comment:
I would use `Sequence[str]` for `execution_names`, otherwise it is unclear what should be passed as input.